src/mockServiceWorker.js

```diff
@@ -99,6 +116,7 @@ self.addEventListener('fetch', function (event) {
 
   event.respondWith(
     new Promise(async (resolve, reject) => {
+      const clientId = sharedClientId || event.clientId
```

Comment:
This value can be computed outside of the `event.respondWith` scope, at the upper scope of the `fetch` event handler. Would you mind moving it up?

Reply:
Yep, I'll do this